resources/user-agents/browsers/sleipnir/sleipnir-ios-1-x.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
I broke these Sleipnir versions into separate files due to the minimum iOS version bump that the major versions had (based on archive.org captures of the iTunes Store page for this app).